build/style_test.go

```diff
@@ -706,7 +687,10 @@ func TestStyle(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if err := stream.ForEach(filter, func(s string) {
+		if err := stream.ForEach(stream.Sequence(
+			filter,
+			stream.GrepNot(`: (field no|type No)Copy is unused \(U1000\)$`),
```

Comment:
@tamird by virtue of using the same linter infrastructure as gosimple and staticcheck, unused now supports the `-ignore` flag. you should be able to ignore the check U1000 in just that one file.